lib/dul_hydra/ability_definitions/mets_folder_ability_definitions.rb

```diff
@@ -0,0 +1,12 @@
+module DulHydra
+  class METSFolderAbilityDefinitions < Ddr::Auth::AbilityDefinitions
+
+    def call
+      can :create, METSFolder do |mf|
+        can? :edit, Collection.find(mf.collection_id)
```

Comment:
Just use `mf.collection_id` here.  The ability logic handles strings - assumes they are object IDs.